local-api/generator_custom_templates/PreFormApi.py

```diff
@@ -52,12 +52,11 @@ def output_reader(proc, outq):
                     preformApi.server_process = server_process
                     return preformApi
                 if "address is already in use" in line:
-                    raise RuntimeError("Port already in use, probably another PreForm server is already running.")
+                    raise RuntimeError(f"Port {preform_port} already in use, probably another PreForm server is already running.")
```

Comment:
Grammar nit inbound: `f"Port {preform_port} is already in use. There's probably another PreForm server already running."`